torch/nn/init.py

```diff
@@ -213,13 +213,20 @@ def _calculate_fan_in_and_fan_out(tensor):
         receptive_field_size = 1
         if tensor.dim() > 2:
             receptive_field_size = tensor[0][0].numel()
-        fan_in = num_input_fmaps * receptive_field_size
-        fan_out = num_output_fmaps * receptive_field_size
+        if is_depthwise is None and num_input_fmaps == 1:
```

Comment:
I'm not sure about this warning.
It will be triggered by anyone using a greyscale dataset while it is irrelevant for them.
Why not make the default `is_depthwise=False` in all the functions?